### PR TITLE
feat: symlink example

### PR DESCRIPTION
**Pull request by lino-levan**

### CODE REVIEW

File: data/symlinks.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+// Now we can create a soft link to the file
+await Deno.symlink("example.txt", "link");
+
+// To resolve the path of a symlink, we can use Deno.realPath
+console.log(await Deno.realPath("link"));
+
+// Symlinks are automatically resolved, so we can just read
+// them like text files
+console.log(await Deno.readTextFileSync("link"));
```
Comment (kt3k) [inline data/symlinks.ts:24]: I think we should remove either `await` or `Sync`

File: data/symlinks.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+// In certain cases, soft links don't work. In this case we
+// can choose to make "hard links".
+await Deno.link("example.txt", "hardlink");
+console.log(await Deno.readTextFileSync("hardlink"));
```
Comment (kt3k) [inline data/symlinks.ts:29]: ditto

File: data/symlinks.ts (hunk 1 above)
Comment (lino-levan) [reply data/symlinks.ts]: Nice catch! Not sure how I missed that.

**Review by kt3k — APPROVED** [review]
LGTM 👍

**Pull request merged**